hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.js

```diff
@@ -117,7 +117,16 @@ export const UnconnectedCheckedOutTable = ({ items, isVisible, toggleVisibility
                                         </IconButton>
                                     </TableCell>
                                     <TableCell align="right">{row.qty}</TableCell>
-                                    <TableCell className={styles.widthBuffer} />
+                                    <TableCell
+                                        align="right"
+                                        className={styles.widthButton}
+                                    >
+                                        <Link to={"/IncidentForm"}>
```

Comment:
We can't just link them to there, we also need to pass information about which item they are reporting. My suggestion would be something like:
- We have somewhere in the store that stores an incident report
- When this button is clicked, dispatch an action to the store that sets the relevant item, then pushes to the incident form
- If the incident form page is loaded and that value in the store isn't set, send them back to the dashboard

You can leave as is for now, just want to call out that this will change

Reply:
In preparation for that change, I would recommend passing in a prop `handleIncidentReport` or something along those lines that takes in an id (which you can get from `row.id` I believe), and making this a button that calls that. Down the line that would be a redux action / thunk that gets passed in.

In the tests, you can then pass in a spy for that prop and see that it gets called when one of the report broken/lost buttons is clicked.

Reply:
Actually, to add to this, I would also pass in `push` as a prop, which the connected component should get in `mapDispatchToProprs` from `connected-react-router`. The `onClick` for the button that goes here should call `reportIncident` (which will be a regular redux action, not a thunk), and then will call `push("/incidentreport")` or whatever the page is called.

The alternative would be to make a `handleIncidentReport` thunk that calls `push`, but that's extra work and testing for the same behaviour.